invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/UserDetails/components/ItemsSearch/components/ResultsList/ResultsList.js

```diff
@@ -15,33 +15,86 @@ export class ResultsList extends Component {
     this.fetchPatronCurrentLoans = this.props.fetchPatronCurrentLoans;
   }
 
+  state = { open: false };
+
+  open_modal = () => () => {
+    this.setState({ open: true });
+  };
+
+  close_modal = () => this.setState({ open: false });
+
   actions(item, itemState) {
+    const buttonCheckout = (
+      <Button
+        content={'Checkout'}
+        onClick={() => {
+          this.checkoutItem(item, this.props.patron).then(() => {
```

Comment:
Perhaps move the click handler to its own instance method for better readability?

Reply:
👍  we started the convention to add something like onButtonClickHandler method in the component